martin-mbtiles/src/tile_copier.rs

```diff
@@ -146,42 +163,64 @@ impl TileCopier {
             .execute(&mut conn)
             .await?;
 
-        if force_simple {
-            for statement in &["CREATE TABLE metadata (name text, value text);",
-                "CREATE TABLE tiles (zoom_level integer, tile_column integer, tile_row integer, tile_data blob);",
-                "CREATE UNIQUE INDEX name on metadata (name);",
-                "CREATE UNIQUE INDEX tile_index on tiles (zoom_level, tile_column, tile_row);"] {
-                query(statement).execute(&mut conn).await?;
-            }
+        if is_empty {
+            if force_simple {
+                for statement in &["CREATE TABLE metadata (name text, value text);",
```

Comment:
why not use the syntax as described in https://www.sqlitetutorial.net/sqlite-primary-key/ ?

Reply:
After our conversation yesterday I looked into it more and realized that `CREATE UNIQUE INDEX ....` doesn't just create an index with non-duplicated keys like I thought, but it _enforces_ a unique constraint on those columns that are part of the key. A unique index also automatically gets generated on the primary key. So that the unique constraint enforced by a unique index is enough for our case; `(zoom_level, tile_column, tile_row)` doesn't have to be the primary key.

Reply:
My mistake, I thought you were referring to the `check_uniqueness` part of the code. When creating a new table, I agree that we can simply create a primary key. I can change this but I don't think it is within the scope of this PR since this is for copying existing mbtiles. I will create a small follow-up PR.